### PR TITLE
AIRFLOW-499 This pull request adds operators for JIRA and Statuspage

### PR DESCRIPTION
**Pull request by aedoran**
Dear Airflow Maintainers,

Please accept this PR that addresses the following issues:
- https://issues.apache.org/jira/browse/AIRFLOW-499

Per Apache guidelines you need to create a [Jira issue](https://issues.apache.org/jira/browse/AIRFLOW/).

Testing Done:
- Unittests are required, if you do not include new unit tests please
  specify why you think this is not required. We like to improve our
  coverage so a non existing test is even a better reason to include one.

Reminders for contributors (REQUIRED!):
- Your PR's title must reference an issue on 
  [Airflow's JIRA](https://issues.apache.org/jira/browse/AIRFLOW/). 
  For example, a PR called "[AIRFLOW-1] My Amazing PR" would close JIRA 
  issue #1. Please open a new issue if required!
- Please squash your commits when possible and follow the [How to write a good git commit message](http://chris.beams.io/posts/git-commit/). 
  Summarized as follows:
  1. Separate subject from body with a blank line
  2. Limit the subject line to 50 characters
  3. Do not end the subject line with a period
  4. Use the imperative mood in the subject line (add, not adding)
  5. Wrap the body at 72 characters
  6. Use the body to explain what and why vs. how

### CODE REVIEW

File: airflow/contrib/operators/jira_operator.py [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+    from airflow.exceptions import AirflowException
+import json
+import logging
+import requests
+import base64
+
+class JIRATransitionOperator(BaseOperator):
+    """
+    This operator is the allows updating a StatusPage Component via the api.
+    docs for it are here https://doers.statuspage.io/api/v1/components/
```
Comment (artwr) [inline airflow/contrib/operators/jira_operator.py:16]: Copy pasta docstring?

File: airflow/contrib/operators/statuspage_component_operator.py [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+        super(StatusPageComponentOperator, self).__init__(*args, **kwargs)
+        self.apikey = apikey
+        self.pageid = pageid
+        self.endpoint= endpoint
+        self.component_id = component_id
+        self.component_status = component_status
+
+    def execute(self, context):
+
+        url = '{endpoint}{pageid}/components/{component_id}.json'.format(endpoint=self.endpoint, pageid=self.pageid,component_id=self.component_id)
```
Comment (artwr) [inline airflow/contrib/operators/statuspage_component_operator.py:36]: Missing slash between endpoint and pageid maybe?

**Comment by ldct** [comment]
change PR message to "add operators for JIRA and statuspage" or similar (http://chris.beams.io/posts/git-commit/#imperative)

**Comment by r39132** [comment]
@aedoran Please address the comments above. The squash your commit and ping me for a review. We are currently working through the backlog of "old" PRs.... in hopes of closing/merging them. Any PRs that are abandoned (`i.e. no movement in a week`) will be closed.

**Comment by r39132** [comment]
@aedoran Please update this PR within the next 5 days or it will be closed for inactivity as part of the cold case PR cleanup : https://cwiki.apache.org/confluence/display/AIRFLOW/Cold-Case+PR+Resolution

**Comment by jhsenjaliya** [comment]
@aedoran are you planning to complete this ? I need to add JiraOperator along with JiraSensor for my use case and I wonder if I should pick up on this jira to submit my work.

**Comment by r39132** [comment]
@jhsenjaliya since this was closed for inactivity, you are welcome to take the submitters' code, add your own, open a fresh PR and submit it for PR review under your name.  In the end, we will want the submitter to respond to feedback and make any necessary changes needed.

**Comment by jhsenjaliya** [comment]
my design for Jira Operator is different and has more depth & detail, I will assign ticket to myself and will submit new patch, Thanks !

**Comment by jhsenjaliya** [comment]
@r39132 , I have another PR (#1950) for adding jira suppport, please take a look when you get a chance   Thanks

**Pull request closed without merging**